### PR TITLE
Doc:Add attribute for kafka client versions

### PR DESCRIPTION
**Pull request by karenzone**
Replaces hard-coded kafka client version with an attribute to make updating it easier

### CODE REVIEW

File: docs/input-kafka.asciidoc [hunk 1]
```diff
@@ -59,7 +66,11 @@ The following metadata from Kafka broker are added under the `[@metadata]` field
 * `[@metadata][kafka][partition]`: Partition info for this message.
 * `[@metadata][kafka][offset]`: Original record offset for this message.
 * `[@metadata][kafka][key]`: Record key, if any.
-* `[@metadata][kafka][timestamp]`: Timestamp in the Record. Depending on your broker configuration, this can be either when the record was created (default) or when it was received by the broker. See more about property log.message.timestamp.type at https://kafka.apache.org/10/documentation.html#brokerconfigs
+* `[@metadata][kafka][timestamp]`: Timestamp in the Record. 
+Depending on your broker configuration, this can be
+either when the record was created (default) or when it was received by the
+broker. See more about property log.message.timestamp.type at
+https://kafka.apache.org/{kafka_client_doc}/documentation.html#brokerconfigs
```
Comment (karenzone) [inline docs/input-kafka.asciidoc:73]: Yikes!  Here's an oldie!

File: docs/output-kafka.asciidoc [hunk 2]
```diff
@@ -328,9 +340,9 @@ Kafka down, etc).
 A value less than zero is a configuration error.
 
 Starting with version 10.5.0, this plugin will only retry exceptions that are a subclass of
-https://kafka.apache.org/25/javadoc/org/apache/kafka/common/errors/RetriableException.html[RetriableException]
+https://kafka.apache.org/{kafka_client_doc}/javadoc/org/apache/kafka/common/errors/RetriableException.html[RetriableException]
 and
-https://kafka.apache.org/25/javadoc/org/apache/kafka/common/errors/InterruptException.html[InterruptException].
+https://kafka.apache.org/{kafka_client_doc}/javadoc/org/apache/kafka/common/errors/InterruptException.html[InterruptException].
```
Comment (karenzone) [inline docs/output-kafka.asciidoc:345]: This link using `/25/` is from a recent PR.  Seems like we should standardize it, too. WDYT?

**Comment by karenzone** [comment]
@kares  I replaced the hard-coded values for kafka client and the kafka version in links with attributes.  The goal is to have one place to set the values in each file. Then all of the other references will be updated automatically.

File: docs/output-kafka.asciidoc (hunk 2 above)
Comment (kares) [reply docs/output-kafka.asciidoc:345]: :+1: most definitely - for correctness these should point to `/24` till we upgrade the client

**Comment by karenzone** [comment]
Thanks for the review, kares! <3

**Comment by karenzone** [comment]
This PR is included in changelog/version bump for #47

**Pull request merged**